script/Ciel/Day 5/05_01_CIEL05_4C.txt

```diff
@@ -197,55 +209,57 @@ Your shoulder is... The bone isn't broken. Good, that means even I should be abl
 [sha:3625ce5439ad3315e343260ca99e5adcb84483ba]{
 -- Page 781, Offset 12974.
 -- 「はい……？　あの、遠野くん？　もしもし……？」
-"Yes...? Umm, Tohno-kun? Can you hear me...?"
+"Hello...? Umm, Tohno-kun? Can you hear me...?"
 }
 [sha:ec52fe1f422324eace45e44bdf8ed653afbe9114]{
 -- Page 781, Offset 12975.
 -- 先輩がいてくれるなら何も怖くはないですよ。
-As long as Senpai is beside me, I'm not afraid of anything.
+As long as Senpai is here beside me, I'm not afraid of anything.
 }
 [sha:783b30d0eebf878e62f712ca4f73b92fee8fd2d0]{
 -- Page 781, Offset 12976.
 -- ……そんな言葉を口にした気になって目蓋を閉じていく。
-...I feel like those words slipped out of my mouth, and I begin to close my eyes.
+Feeling like those words may have slipped out of my mouth, I begin to close my eyes...
 }
 [sha:7e5c0c4cf421014ed52a34e2b4b3d4933035b9d6]{
 -- Page 782, Offset 12977.
 -- 視界に映るものはあの月だけ。それで、ここにはいないもうひとりの協力者の顔を思い浮かべた。
-The only thing I can still see is the moon. And I'm reminded of another person who helped but isn't here.
+The only thing I can still see is the moon.
+ It calls to mind the face of someone else, who helped but isn't here.
 }
 [sha:cd80ef39be9fc7eb1c5d94a35d76138e812e50c7]{
 -- Page 782, Offset 12978.
 -- アルクェイド。あいつに会ったらなんて言おう。
-Arcueid. What am I going to say when I see her again.
+Arcueid. What am I going to say when I see her again?
 }
 [sha:2302d79ae9a6fb497bd30722898ca7ab9e1f5d2a]{
 -- Page 782, Offset 12979.
 -- 話が違うと抗議するべきか、ありがとうとお礼を言うべきか。
-Should I protest that she didn't do what she said she would? Or should I thank her?
+Should I complain that she went back on her word? Or should I thank her?
 }
 [sha:addd1c56f9cbfbcfa92b02406d0f22bb2f221558]{
 -- Page 782, Offset 12980.
 -- どっちにしても、あいつがどんな顔をするのか想像するのは、なんていうか―――
-Either way, I imagine what kind of face she'll make, and I―――
+Either way, just thinking about what kind of face she'll make is, how should I put it―――
 }
 [sha:a75529e2fa75b8527e8893b3eb43549ebe185966]{
 -- Page 783, Offset 12981.
 -- 「……はい、おやすみなさい遠野くん。
-"...Yes, good night Tohno-kun.
+"Well then... Goodnight, Tohno-kun.
 }
 [sha:db581565f98dbd1a917824836a0301f06768c8fc]{
 -- Page 783, Offset 12982.
 -- 後の事はわたしに任せて、どうかゆっくり休んでください」
-Leave the rest to me and rest well."
+Leave the rest to me, and sleep tight."
 }
 [sha:6f45a90664d313ba72fd70431dfe8f94d778e5b7]{
 -- Page 784, Offset 12983.
 -- 最後に聞こえたのは子守歌のような囁きだけ。
-The last thing I heard was a whisper like a lullaby.
+The last thing I hear is a soft lullaby of a whisper.
 }
 [sha:04514c1266e7d245c3d62e45cbac4e60f70951b1]{
 -- Page 784, Offset 12984.
 -- 俺の意識は二日ぶりに、深い眠りに落ちていった。
-I fall into a deep slumber, the first in two days.
+// Tsukihime just the story of a man who doesn't get any goddamn sleep
```

Comment:
post "sleep tight shiki" on this post in 5 minutes for a full life of "I fall into a deep, dark sleep"